From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Mon, 12 Aug 2019 12:44:27 -0700
Subject: [PATCH] Fixups for packaging and stateless config

- The config subpackage disappeared, so update buildreq_add
- Add stateless.patch to enable CMake to find the new config located at
  /usr/share/defaults/etc/ldc2.conf
- Update SYSCONF_INSTALL_DIR and refresh install_append

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 driver/configfile.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/driver/configfile.cpp b/driver/configfile.cpp
index cac8c78..c92e6d0 100644
--- a/driver/configfile.cpp
+++ b/driver/configfile.cpp
@@ -157,6 +157,10 @@ bool ConfigFile::locate(std::string &pathstr) {
   // try /etc/ldc (absolute path)
   p = "/etc/ldc";
   APPEND_FILENAME_AND_RETURN_IF_EXISTS
+
+  // try /usr/share/defaults/etc (stateless path)
+  p = "/usr/share/defaults/etc";
+  APPEND_FILENAME_AND_RETURN_IF_EXISTS
 #endif
 
 #undef APPEND_FILENAME_AND_RETURN_IF_EXISTS
